.github/workflows/quality.yaml

```diff
@@ -4,21 +4,13 @@ on:
 jobs:
   python_quality_check:
     name: Run Python quality checks
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

Comment:
not against running it on macos, esp if it gets the problem solved, could be a good idea, especially for linting stuff, to have it run on several machines to make sure it is compatible for dev